### PR TITLE
Improve logs

### PR DESCRIPTION
**Pull request by inzamam-iqbal**
Use native python logger library for logging
use --versbose option to set the logger to debug level, default is warning

fixes: #27

### CODE REVIEW

**Comment by inzamam-iqbal** [comment]
@nicolapaoli @alexvicegrab snyk is complaining about urllib3 which is not one of our direct dependency. We depend on sphinx 1.8.4 which is the latest version. But sphinx depends on request as `requests>=2.5.0`.  request depend on urllib3 as `urllib3>=1.21.1,<1.26`.  
So we can't directly solve it. Also the way sphinx depends on request may break sphinx when request get updated to 3.0  
So what are we going to do about this?

File: nephos/helpers/misc.py [hunk 1]
```diff
@@ -17,43 +18,41 @@
 t = Terminal()
 
 
-def execute(command, verbose=False, show_command=True, show_errors=True):
+def execute(command, show_response=True, show_command=True, show_errors=True):
```
Comment (dbandin) [inline nephos/helpers/misc.py]: Having the logging module now, can't we remove show_response, show_command and show_errors and the logic for those flags, and just print the different levels of logging, will make this method quite more compact and clear.

File: nephos/helpers/misc.py [hunk 2]
```diff
@@ -65,16 +64,15 @@ def execute_until_success(command, verbose=False, delay=15):
         res, err = execute(
             command,
             show_command=first_pass,
-            verbose=verbose and first_pass,
+            show_response=first_pass,
```
Comment (dbandin) [inline nephos/helpers/misc.py]: This wont be needed if we remove the arguments from the execute as stated in the previous comment.

**Comment by dbandin** [comment]
@inzamam-iqbal Seems like Sphinx==1.8.5 is available in the repo, the next availables are major versions upgrade, jumps to 2.x.x. The version 1.8.5 jumps urllib3 to 1.25.3, it may be clear, otherwise we need to move to a major version.
twine and kubernetes are also depending in this urllib vulnerable version. Maybe we can ignore this snyk issue and create another issue for the dependency upgrades.

**Comment by inzamam-iqbal** [comment]
@dbandin Thank you for the feedbacks
I will add the option --out / -o to set the output file for logs currently I don't send it to a log file
Currently I set log level based on --verbose option if you set the --verbose option log level is set to DEBUG if not to INFO
All setting of logger is applied at deploy.py file
We know all modules use that modified logger because I modify the root logger
Regarding the execute function, I just preserved the same nature as it before. Now I will change it according to your suggestion.

**Comment by dbandin** [comment]
@inzamam-iqbal sounds good to me!, thanks for addressing it.

**Comment by inzamam-iqbal** [comment]
Here is an image of log file created 

logs

**Comment by alexvicegrab** [comment]
As @dbandin, I would ignore Snyk and merge. I'd migrate to Dependabot, which is free, tightly integrated with GitHub and auto-creates PRs that can be easily tested and merged. We may need @ryjones to help add the Dependabot plugin.

**Comment by alexvicegrab** [comment]
Sphinx is needed for documentation. And it should be safe to upgrade to major version 2, and then test that the documentation is correctly created. I recommend this link for documentation explanation (it's what I used in the first place to create it):

https://samnicholls.net/2016/06/15/how-to-sphinx-readthedocs/

**Comment by alexvicegrab** [comment]
Regarding a specialised Logger module, I'd relegate that to a separate PR, so we build progressively. But agree on the current suggestions for the `show_errors`, `show_command`, etc.

**Comment by dbandin** [comment]
Seems like there is an error with the integrations tests, @inzamam-iqbal can you verify that before the merge?

**Comment by inzamam-iqbal** [comment]
@dbandin now integration test passes.

**Pull request merged**